<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#backgroundWidget { background-color: #F5F6F8; }</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QWidget" name="container" native="true">
        <property name="styleSheet">
         <string notr="true">#container { background-color: #FFFFFF; border-radius: 10px; }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="btnBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#btnBox{ border:none; border-bottom: 2px solid #F5F6F8 ; }</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="checkBtn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#checkBtn { 
	background-color: #FFFFFF; 
	font-size: 14px; 
	color: #FF6699; 
	border:none; 
	border-bottom: 2px solid #FF6699 ; 
	font-family:微软雅⿊; 
	font-weight:bold; 
}
</string>
              </property>
              <property name="text">
               <string>审核管理</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="roleBtn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>60</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#roleBtn { 	
	background-color: #FFFFFF; 
	font-size: 14px; 
	color: #666666; 
	border:none;
	border-bottom: 2px solid #F5F6F8; 
	font-family:微软雅⿊; 
}</string>
              </property>
              <property name="text">
               <string>角色管理</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1067</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="pageBox" native="true">
           <widget class="CheckTable" name="checkTable" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>686</height>
             </rect>
            </property>
           </widget>
           <widget class="RoleTable" name="roleTable" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1270</width>
              <height>686</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckTable</class>
   <extends>QWidget</extends>
   <header location="global">checktable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoleTable</class>
   <extends>QWidget</extends>
   <header location="global">roletable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
